src/client.rs

```diff
@@ -55,14 +73,90 @@ impl SpiceClient {
         })
     }
 
+    /// Queries the Spice Flight endpoint with the given SQL query.
+    /// ```
+    /// use spice_rs::Client;
+    ///
+    /// #[tokio::main]
+    /// async fn main() {
+    ///   let mut client = Client::new("API_KEY").await.unwrap();
```

Comment:
For all methods after `new`, I wouldn't include the `let mut client = Client::new("API_KEY").await.unwrap();` line. Users should be able to just see the `::new(...)` method (and are expecting such a method).